pom.xml

```diff
@@ -98,7 +98,7 @@
         <url>https://github.com/opendistro-for-elasticsearch/security</url>
         <connection>scm:git:git@github.com:opendistro-for-elasticsearch/security.git</connection>
         <developerConnection>scm:git:git@github.com:opendistro-for-elasticsearch/security.git</developerConnection>
-        <tag>1.4.0.1</tag>
+        <tag>1.4.0.2</tag>
```

Comment:
Consider `${project.version}`. Is `tag` used by the build?

Reply:
Looks like scm:tag is not correctly used.  Looking up online it seems to be using HEAD as common value. 
Maybe when the security repos were separate, this was used to push to local mvn repository and used for building there. 

Maybe investigate this and put the correct value separately as `${project.version}` may not be correct as well